content/en/docs/Badges/MCH2022/software-development/fpga/_index.md

```diff
@@ -1,20 +1,32 @@
 ---
-title: "Getting Started with FPGA Development"
+title: "FPGA Development"
 linkTitle: "FPGA"
 nodateline: true
 weight: 1
 ---
 
-# FPGA development
+## TL;DR
 
-The badge contains an ice40 FPGA that is connected to a PMOD connector, some
-RAM, and an RGB LED.  It can also control the display over a parallel bus, and
-has an USB UART link via the RP2040 and an SPI link to the ESP32. See here for
-a confusing [diagram with boxes and arrows](../../hardware) showing how everything fits together.
+```
+git clone https://github.com/badgeteam/mch2022-tools/
+git clone --recursive https://github.com/badgeteam/mch2022-firmware-ice40
+python3 mch2022-tools/webusb_fpga.py mch2022-firmware-ice40/projects/Hello-World/hello_world.bin
+```
 
-Check out our ["FPGA **Getting Started** for Badgers with Tiny
```

Comment:
Why remove this link? Purposely or oversight?

Reply:
Ah, I see you moved it way to the bottom ... I think it (mainly because I wrote it :) it should probably be higher up, else people won't notice. May a: "if the TLDR wasn't wordy enough for you, try ..."

Reply:
Next commit handles that!

Reply:
The badgers intro needs some more attention, for sure. The projects are described now in the file I took care of.